homeassistant/components/sensor/launch.py

```diff
@@ -12,15 +13,19 @@
 from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME
 from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.util import Throttle
 
-REQUIREMENTS = ['pylaunches==0.0.2']
+REQUIREMENTS = ['pylaunches==0.1.1']
 
 _LOGGER = logging.getLogger(__name__)
 
 CONF_ATTRIBUTION = "Data provided by Launch Library."
 
 DEFAULT_NAME = 'Launch'
 
+TIME_BETWEEN_UPDATES = timedelta(hours=1)
```

Comment:
Why not just set a higher `SCAN_INTERVAL` ?

Reply:
Could be any number of reasons, the wind direction, solar flares, sloppy copy+paste... :D 

Have changed it to `SCAN_INTERVAL` now :)

Reply:
LOL, that was an honest question. I see that pattern in several places and was wondering if I was missing something.

You actually don't need `LaunchData` now but could just pass the `api`. Up to you ...

Reply:
Aha :) That makes sense :)